cli/replicaset/rebootstrap.go

```diff
@@ -0,0 +1,109 @@
+package replicaset
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/running"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// RebootstrapCtx rebootstrap operations arguments.
+type RebootstrapCtx struct {
+	// AppName application name the instance belongs to.
+	AppName string
+	// InstanceName is an instance name to re-bootstrap.
+	InstanceName string
+	// Confirmed is true if re-bootstrap confirmation is not required.
+	Confirmed bool
+}
+
+// cleanDataFiles removes snap, xlog and vinyl artifacts.
+func cleanDataFiles(instCtx running.InstanceCtx) error {
+	filesToRemove := []string{}
+	for _, pattern := range [...]string{
+		filepath.Join(instCtx.MemtxDir, "*.snap"),
+		filepath.Join(instCtx.WalDir, "*.xlog"),
+		filepath.Join(instCtx.VinylDir, "*.vinyl"),
+	} {
+		if foundFiles, err := filepath.Glob(pattern); err != nil {
+			return err
+		} else {
+			filesToRemove = append(filesToRemove, foundFiles...)
+		}
+	}
+
+	for _, fileToRemove := range filesToRemove {
+		stat, err := os.Stat(fileToRemove)
+		if err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			return fmt.Errorf("cannot get info of %q: %s", fileToRemove, err)
+		}
+		if stat.Mode().IsRegular() {
+			if err = os.Remove(fileToRemove); err != nil {
+				return fmt.Errorf("cannot remove %q: %s", fileToRemove, err)
```

Comment:
```suggestion
				return fmt.Errorf("cannot remove %q: %w", fileToRemove, err)
```

Reply:
Why wrapping?

Reply:
Well, if we don’t need to do unwrap now, this doesn’t mean that it will never be needed. In addition, there is less confusion when errors in the project are handled in a similar way.

On the other hand, there are no downsides at it.

Reply:
A little bit confusing for me, but if so - up to you.